api/logs/logSender.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/gorilla/websocket"
 )
 
+const disconnectMessage = -1
```

Comment:
instead of this const might check if there is a const in websocket package (like websocket.CloseMessage)

Reply:
the constant you mention is unexported so it can't be used

Reply:
ok